sim/src/main/scala/midasexamples/AutoCounterModule.scala

```diff
@@ -71,6 +71,15 @@ class AutoCounterModuleChild extends MultiIOModule {
   io.oddlfsr := odd_lfsr
 }
 
+/** Demonstrate explicit instrumentation of AutoCounters via PerfCounter 
+ *
+ * Toplevel Chisel class suitable for use as a GoldenGate 'target' as described by the docs
+ *
+ * @see https://docs.fires.im/en/latest/Advanced-Usage/Debugging-and-Profiling-on-FPGA/AutoCounter.html#ad-hoc-performance-counters
+ * @see https://docs.fires.im/en/latest/Advanced-Usage/Generating-Different-Targets.html
+ * @see AutoCounterF1Test
```

Comment:
Do you guys publish the rendered ScalaDoc somewhere?  I'm curious about how these `@see` links will render but I'm pretty sure that having this here would have let me grok the testiness of what you were doing more quickly.

Reply:
You can do it from sbt to build it yourself.  If you want to see the published stuff:
https://fires.im/firesim/latest/api/
Or for the release: 
https://fires.im/firesim/1.11.0/api/

Reply:
Thanks for the pointer.  I was being lazy and not going to look for how they get published in the circleci stuff or trying to remember how to regen them locally.  I didn't see a link to the published scaladoc from either https://fires.im or https://docs.fires.im.

Reply:
Where do you think is an appropriate place. You think we should just put the link in the front page of the docs? 
I think the only slight annoyance is getting the url to point to the right version. I'm sure there's a RST macro (or something) for that available in readthedocs.